lib/v2/screens/dashboard/interactor/viewmodels/wallet_event.dart

```diff
@@ -5,12 +5,19 @@ import 'package:meta/meta.dart';
 @immutable
 abstract class WalletEvent extends Equatable {
   const WalletEvent();
+
   @override
   List<Object> get props => [];
 }
 
 class RefreshDataEvent extends WalletEvent {
   const RefreshDataEvent();
+
   @override
   String toString() => 'RefreshDataEvent';
-}
\ No newline at end of file
+}
+
+class LoadUserValues extends WalletEvent {
+  @override
+  String toString() => 'LoadProfileValues';
+}
```

Comment:
I renamed this event in a previous PR, please use that name. It makes more sense to see it in the logs.

Reply:
is now called LoadWalletData

Reply:
yes we want to know from were is a event fired and what is doing.

Reply:
could be OnLoadWalletData too.

Reply:
that the name forgot it had the On